includes/class-sensei-course.php

```diff
@@ -3889,13 +3889,13 @@ public static function alter_redirect_url_after_enrolment( $url, $post ) {
 	 * @since 4.2.0
 	 * @access private
 	 *
-	 * @return int Total days to completion, rounded to the highest integer.
+	 * @return float Total days to completion, rounded to the highest integer.
 	 */
 	public function get_days_to_completion_total() {
```

Comment:
Do you think the method name could be improved? I'm not sure if having the `total` word makes sense here. Maybe `get_average_days_to_completion`?

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/5097/commits/6c3729f7909fe193851338d96b10c2587a10b649